.github/workflows/upload-drivers.yml

```diff
@@ -24,9 +28,29 @@ jobs:
         with:
           credentials_json: '${{ secrets.GOOGLE_CREDENTIALS_COLLECTOR_SVC_ACCT }}'
 
+      - name: 'Set up Cloud SDK'
+        uses: 'google-github-actions/setup-gcloud@v1'
+
       - name: Push drivers
         uses: 'google-github-actions/upload-cloud-storage@v1'
         with:
           path: /tmp/output/
           parent: false
           destination: ${{ inputs.gcp-bucket }}
+
+      - name: Push to merged bucket
+        run: |
+         shopt -s nullglob
+         shopt -s dotglob
+
+          # We use gsutil here because we need to ignore files that already
```

Comment:
It would be nice to describe the overall schema of merging in plain text somewhere. A comment somewhere, or a new paragraph in the driver-build docs?